src/priorityQueue.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+class Node {
+  constructor(follows, data, priority, proceeds) {
+    this.after = follows
+    this.info = data
+    this.priority = priority
+    this.before = proceeds
+  }
+}
+
+class priorityQueue {
+  constructor() {
+    this.head = null
+    this.tail = null
+    this.count = 0
+  }
+}
+
+  enqueue(value, num) {// adds an element with priority (number) to the back of the queue.
+    let behind = this.tail
+    let inFront = null
+    const element = new Node(behind, value, num, inFront)
+    this.count === 0
+      ?(this.tail = element,
```

Comment:
The use of a ternary operator here makes the code incredibly difficult for a human to parse.  I would recommend the infinitely more readable `if`/`else` construct.